library/src/main/java/com/mux/video/upload/internal/UploadMetrics.kt

```diff
@@ -217,7 +217,7 @@ internal class UploadMetrics private constructor() {
         appContext.packageManager.getPackageInfo(appContext.packageName, 0)
       }
       appName = packageInfo.packageName
-      appVersion = packageInfo.versionName
+      appVersion = packageInfo.versionName ?: "unknown"
```

Comment:
question(non-blocking): When would the "nullish coalescing" fallback case (aka expression eval to `"unknown"`) occur? Would this be a bug/breakdown of the build or build config? If yes, should we signal that somewhere/somehow?

Reply:
I *think* this is a [built-in API](https://developer.android.com/reference/android/content/pm/PackageInfo#versionName). From the docs:

> The version name of this package, as specified by the <manifest> tag's [versionName](https://developer.android.com/reference/android/R.styleable#AndroidManifest_versionName) attribute, or null if there was none.

Reply:
Yep! If a consumer app didn't declare a version, now we get 'unknown' for a label instead of 'null'